packages/shared-react/src/api/queryClient.ts

```diff
@@ -2,7 +2,7 @@ import { FIVE_MINUTES_MS, ONE_MINUTE_MS } from '@alephium/shared'
 import { QueryClient, QueryClientConfig } from '@tanstack/react-query'
 import { AxiosError } from 'axios'
 
-const MAX_RETRIES = 3
+const MAX_RETRIES = 10
```

Comment:
Since our API produces many 429's now, we want to avoid having incomplete data because all 3 retries have failed